packages/rum-core/src/domain/rumEventsCollection/action/trackActions.spec.ts

```diff
@@ -121,6 +126,16 @@ describe('trackActions', () => {
     expect(discardSpy).toHaveBeenCalled()
   })
 
+  it('discards a pending action with a negative duration', () => {
+    const { domMutationObservable, clock } = setupBuilder.build()
+    mockValidatedClickAction(domMutationObservable, clock, button, -BEFORE_PAGE_ACTIVITY_VALIDATION_DELAY)
```

Comment:
```suggestion
    mockValidatedClickAction(domMutationObservable, clock, button, -1)
```
or is there any reason to use this constant here?